systemtests/src/main/java/io/enmasse/systemtest/scale/ResultWriter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.systemtest.scale;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.Map;
+
+public class ResultWriter {
```

Comment:
@lulf so this can be used in scale test as well, right? Or do we have any similiar class already (so you can reuse it)?

Reply:
In the scale test it was kind of built into the test class, so I can refactor it to use this as well.

Reply:
Actually, when reading the code, it seems it does not write the same data to result.json and plot data, so I'm not sure how useful this class will be there.